mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/BrukerRoutes.kt.kt

```diff
@@ -37,11 +43,34 @@ fun Route.brukerRoutes() {
                 status = HttpStatusCode.BadRequest
             )
             val accessToken = call.getAccessToken()
-            historikkService.hentHistorikkForBruker(fnr, accessToken)?.let { call.respond(it) }
-                ?: call.respondText(
-                    "Klarte ikke hente historikk for bruker",
-                    status = HttpStatusCode.InternalServerError
-                )
+            auditLog.log(createAuditMessage("NAV-ansatt med ident: '${getNavIdent()}' forsøker å se på tiltakshistorikken for bruker med ident: '${getNorskIdent()}'."))
+            historikkService.hentHistorikkForBruker(fnr, accessToken)?.let {
+                auditLog.log(createAuditMessage("NAV-ansatt med ident: '${getNavIdent()}' har sett på tiltakshistorikken for bruker med ident: '${getNorskIdent()}'."))
+                call.respond(it)
+            }
+                ?: run {
+                    auditLog.log(createAuditMessage("NAV-ansatt med ident: '${getNavIdent()}' fikk ikke sett på tiltakshistorikken for bruker med ident: '${getNorskIdent()}'."))
+                    call.respondText(
+                        "Klarte ikke hente historikk for bruker",
+                        status = HttpStatusCode.InternalServerError
+                    )
+                }
         }
     }
 }
+
+private fun PipelineContext<Unit, ApplicationCall>.createAuditMessage(msg: String): CefMessage? {
+    return CefMessage.builder()
+        .applicationName("mulighetsrommet-api")
+        .event(CefMessageEvent.ACCESS)
+        .name("Arbeidsmarkedstiltak")
```

Comment:
Hva brukes dette navnet til?

Reply:
Vet ikke mer enn at det dukker opp i den CEF-formaterte logg-strengen. Min tanke var at det er et mer lesbart navn for hele systemet som en enhet, enn hver enkelt del (api, adapter, osv...)